xml/cap_depl_about.xml

```diff
@@ -225,5 +227,90 @@
    and Developing Buildpacks</link> to learn more about buildpacks, and
    creating and managing your own.)
   </para>
+
+  <para>
+   The following figures illustrate the main components of &productname;.
+   <xref linkend="fig.cap.cloud-101"/> shows a comparison of the basic cloud
+   platforms: Infrastructure as a Service (IaaS), Container as a Service
+   (CaaS), Platform as a Service (Paas), and Software as a Service (SaaS).
+   &suse; &caasp; is a Container as a Service platform, and &productname; is
+   Platform as a Service.
```

Comment:
is a Platform as a Service

Reply:
Thanks, I was trying to not say 'platform' many times in one sentence.